tests/test_depth.py

```diff
@@ -0,0 +1,81 @@
+import appdirs
+import pandas as pd
+from pathlib import Path
+from pydantic import ValidationError
+import pytest
+
+from zamba.models.depth_estimation import DepthEstimationManager, DepthEstimationConfig
+
+from conftest import TEST_VIDEOS_DIR
+
+
+@pytest.fixture
+def two_video_filepaths(tmp_path):
+    output_path = tmp_path / "filepaths.csv"
+
+    # from labels csv
+    filepaths = [
+        str(
+            TEST_VIDEOS_DIR
+            / "data/raw/goualougo_2013/gorillas_2013/MPI_FID_20_Duboscia/14-Jun-2013/FID_20_Duboscia_2013-6-14_0083.AVI"
+        ),
+        str(TEST_VIDEOS_DIR / "data/raw/savanna/Grumeti_Tanzania/K38_check3/09190048_Hyena.AVI"),
+    ]
+
+    pd.DataFrame(columns=["filepath"], data=filepaths).to_csv(output_path)
+    return output_path
+
+
+def test_prediction(two_video_filepaths):
+    dem = DepthEstimationManager(model_cache_dir=Path(appdirs.user_cache_dir()) / "zamba")
+
+    filepaths = pd.read_csv(two_video_filepaths).filepath.values
+    preds = dem.predict(filepaths)
+
+    assert preds.shape == (77, 3)
+    assert preds.distance.notnull().sum() == 2
+    assert preds.filepath.nunique() == 2
+
+    # two detections for frame 0
+    assert preds.set_index(["filepath", "time"]).loc[filepaths[0], 0].shape[0] == 2
+
```

Comment:
add a check that the distance estimates are the same as what we get through running through the comp winning model (use a video with single animal)